src/evaluator.ts

```diff
@@ -76,9 +80,19 @@ export class Evaluator {
     bucketBy: string,
     percentage: number,
   ): boolean {
-    const property = this.getAttrValue(target, bucketBy);
+    let property = this.getAttrValue(target, bucketBy);
     if (!property) {
-      return false;
+      const old_bucketBy = bucketBy;
```

Comment:
The standard in JS is camel-case, not snake
```suggestion
      const oldBucketBy = bucketBy;
```

Reply:
lol - I work on so many languages these days I've no idea anymore